index.js

```diff
@@ -27,20 +27,14 @@ async function deployProject(run) {
  * @param {function} run - function provided under utils by Netlify to build event functions.
  * @param {string} functionsDir - Path to the actions directory.
  */
-async function deployActions({
-  run,
-  functionsDir,
-  secretsPath,
-  timeout,
-  memory
-}) {
+async function deployActions({run, functionsDir, timeout, memory}) {
   const files = await readdir(functionsDir);
   for (const file of files) {
     const [actionName, extension] = file.split('.');
     let command =
       `nim action update ${actionName} ${join(functionsDir, file)} ` +
       `--timeout=${Number(timeout)} --memory=${Number(memory)} ` +
-      `--web=raw --env-file=${secretsPath} `;
+      `--web=raw `;
```

Comment:
```suggestion
      `--web=raw`;
```

Reply:
does `run.command` re-export `process.env`?

Reply:
Yes. https://github.com/sindresorhus/execa#extendenv

The space is on purpose.

Reply:
> The space is on purpose.

K - a comment in the future will be helpful.